plane/source/docs/guide-tailsitter.rst

```diff
@@ -243,9 +272,9 @@ interpreted using the :ref:`Q_TAILSIT_INPUT<Q_TAILSIT_INPUT>` parameter. The cho
 Tailsitter Input Mask
 =====================
 
-To support people flying 3D aircraft and wanting to learn how to
-prop-hang manually, you can set the :ref:`Q_TAILSIT_MASK<Q_TAILSIT_MASK>` to a mask of
-channels that will have full manual input control while hovering.
+To support people with experience flying 3D aircraft and wanting to learn how to
+prop-hang manually, you can set the :ref:`Q_TAILSIT_MASK<Q_TAILSIT_MASK>` to determine which
+channels that will have full manual input control without attitude stabilization while hovering in QHOVER and QSTABILIZE.
```

Comment:
determine which channels that ~~will~~ have full

Reply:
done